web/src/beta/components/StorytellingPageSectionItem/index.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from "@emotion/styled";
```

Comment:
Don't use styled directly from @emotion/styled. import it from @reearth/services/theme